packages/universal-components/package.json

```diff
@@ -33,6 +33,7 @@
     "@kiwicom/orbit-design-tokens": "^0.5.0",
     "@storybook/react-native": "^4.1.13",
     "memoize-one": "^5.0.0",
+    "react-draggable": "^3.3.0",
```

Comment:
I have some mixed feelings about this library. I understand that it saves some time than doing our implementation.
But it throws some errors into console after drag.
<img width="1249" alt="Screenshot 2019-05-03 at 10 23 34" src="https://user-images.githubusercontent.com/2660330/57126703-2c729080-6d8e-11e9-8f8c-9569f0f7e71c.png">

It looks like related to this issue https://github.com/mzabriskie/react-draggable/issues/390

Reply:
Hmm, that's weird 😕it should be possible to use `DraggableCore` component which is more low-level than `Draggable`. I think it could solve this problem.

Reply:
Should be fixed (with `DraggableCore`).

Reply:
Cool, seems to be ok now. I also like that this lighter component is enough for us. 
That this library is slightly overkill was part of my concern, but I feel better about it now 😉.
Thanks